setup.py

```diff
@@ -14,7 +14,7 @@
 
 INSTALL_REQUIRES = [
     "transformers >= 4.26.0",
-    "optimum",
+    "optimum @ git+https://github.com/huggingface/optimum.git",
```

Comment:
Hmmmm, should we require that?

Reply:
I will revert it. I previously put it this way as the CI will fail without our lazy import fix. I think that we shall install optimum from source for some CIs, but I also agree that we shall not require users to install optimum from source for using it, so maybe in TEST_REQUIRES but not INSTALL_REQUIRES.